.github/scripts/trymerge.py

```diff
@@ -1411,6 +1411,9 @@ def find_matching_merge_rule(
             if rule.ignore_flaky_failures
             else 0,
         )
+        if len(required_checks) == 0:
```

Comment:
```suggestion
        if not required_checks
```

This should be sufficient :)